tfac/figures/figureCommon.py

```diff
@@ -24,15 +25,25 @@
 matplotlib.rcParams["legend.labelspacing"] = 0.2
 matplotlib.rcParams["legend.markerscale"] = 0.7
 matplotlib.rcParams["svg.fonttype"] = 'none'
```

Comment:
Is it this line that's changing the fonts, potentially?

Reply:
svg.fonttype = None is telling it to not convert the font to outline. I think you can end up with a different font when you don't have the one indicated. We could change it to something really common, because I think it sets it to some uncommon open source one. We can also work this out later when the figures are generally laid out.